File: source/files/scripts/异或png/xor.py
#!python
# coding=utf-8
import sys

def xor(filename):
    with open(filename, 'rb') as f, open('xor.png', 'wb') as wf:
        for each in f.read():
            wf.write(each ^ 0xff)


if __name__ == '__main__':
    xor(sys.argv[1])

File: source/files/scripts/异或png/xor.js
#!node

const fs = require('fs');
let out = fs.createWriteStream('xor.png');
let input = fs.createReadStream(process.argv[2]);
input.on('data', data => {
    let chunk = [];
    for (let each of data) chunk.push(each ^ 0xff);
    out.write(Buffer.from(chunk));
});
console.log(process.argv[2] + ' finish');